README.md

```diff
@@ -39,38 +39,40 @@ This vagrant box aims to make it dead simple to start a hashistack and emulate h
 
 ## Content
 
-1. [Description - what & why](#description---what--why)
-    1. [Submodules](#submodules)
-    2. [Services](#services)
-    3. [Why does this exist?](#why-does-this-exist)
-    4. [Installed stack](#installed-stack)
-    5. [Versions](#versions)
-2. [Install prerequisites](#install-prerequisites)
-    1. [General requirements](#general-requirements)
-        1. [Proxy](#proxy)
-    2. [Linux requirements](#linux-requirements)
-    3. [MacOS requirements](#macos-requirements)
-    4. [Windows requirements](#windows-requirements)
-        1. [GNU Make and Git Bash](#GNU-Make-and-Git-Bash)
-        2. [Installation](#installation)
-3. [Build](#build)
-4. [Configuration](#configuration)
-    1. [Default Configuration](#default-configuration)
-    2. [Override default configuration](#override-default-configuration)
-        1. [ENV variables](#option-1---env-variables)
-        2. [Config files](#option-2---config-files)
-5. [Usage](#usage)
-   1. [Starting a plain default box](#option-1-starting-a-plain-default-box)
-      1. [Port collisions](#port-collisions)
-         1. [Shut down the running machine](#shut-down-the-running-machine)
-         2. [Use the `auto_correct` feature to dynamically allocate ports](#use-the-auto-correct-feature-to-dynamically-allocate-ports)
-   2. [Starting a new project based on the template](#option-2-starting-a-new-project-based-on-the-template)
-6. [Test](#test)
-    1. [Local run](#local-run)
-    2. [CI pipeline run](#ci-pipeline-run)
-        1. [CI test configuration](#ci-test-configuration)
-8. [Diagram](#diagram)
-9. [Contribute](#contribute)
```

Comment:
It is quite a big change the way you have formatted the documentation.

Reply:
It was automatic, but done intentionally. 

Like I said last week I think this makes more sense because the documentation is not a step-by-step guide.

Do you think it's too much to do in this PR?

Reply:
I agree that doc is not step-by-step guide. But numeric content makes easier orientation in documentation (IMHO, if others ok to remove numbers, I am ok with it)

Reply:
I also agree that numeric content makes it easier to orientate in the docs.

Reply:
I disagree, but am ok changing it back for now

Reply:
I really don't have one preferred over the other and would be fine either or. What I do believe is that we should aim to keep it consistent with the other readmes' even if it is a step by step guide or doc. And for step by step guides, numbers would be the only choice that could fit and again would fit both scenarios to be fair.

Reply:
That is a fair point. 

BTW: I've changed it back